.github/workflows/testing.continuous-integration.yml

```diff
@@ -0,0 +1,50 @@
+name: Continuous Integration
+
+on:
+    push:
+        branches:
+            - release/live
```

Comment:
So we're calling this a "release" branch?  Not saying it doesn't work, though want to make sure this won't be confusing to anyone.  I do think we don't need `/live`, not sure what that means.